sql/core/src/main/scala/org/apache/spark/sql/internal/CatalogImpl.scala

```diff
@@ -43,12 +43,6 @@ class CatalogImpl(sparkSession: SparkSession) extends Catalog {
 
   private def sessionCatalog: SessionCatalog = sparkSession.sessionState.catalog
 
-  private def requireDatabaseExists(dbName: String): Unit = {
```

Comment:
why inline this function?

Reply:
since it is only used in two places that will both need to change its behavior I went ahead and inline here. I can revert since I'm no longer doing `listFunctions` (the second call site)